universal-application-tool-0.0.1/app/controllers/applicant/ApplicantProgramReviewController.java

```diff
@@ -67,6 +68,12 @@ public ApplicantProgramReviewController(
         .thenApplyAsync(
             (roApplicantProgramService) -> {
               ImmutableList<AnswerData> summaryData = roApplicantProgramService.getSummaryData();
+              int allBlockCount = roApplicantProgramService.getAllBlocks().size();
+              int completedBlockCount =
+                  roApplicantProgramService.getAllBlocks().stream()
+                      .filter(Block::isCompleteWithoutErrors)
+                      .mapToInt(b -> 1)
```

Comment:
😂  Never seen this technique for getting the size from a stream. Nice.

Reply:
the `count` method returns a long and that's annoying.